Classes/Negotiation/DefaultNegotiation.php

```diff
@@ -26,7 +26,10 @@ public function __invoke(NegotiateSiteLanguageEvent $event): void
         foreach ($userLanguages->toArray() as $userLanguage) {
             foreach ($event->getSite()->getLanguages() as $siteLanguage) {
                 foreach ($compareWith as $function) {
-                    if ($siteLanguage->enabled() && (string)$userLanguage === $this->normalizer->normalize((string)$siteLanguage->{$function}())) {
+                    if ($siteLanguage->enabled() && 
+                        $siteLanguage->toArray()['excludeFromLanguageDetection'] !== true &&
```

Comment:
Hey @devmes 
I will add the Null Coalescing Operator shortly, so there is no error, if the setting is missing in the SiteLanguage.
Thank you.

Reply:
Sorry... could you add the Null Coalescing check, so the tests are working?